gsoc/2016.md

```diff
@@ -47,12 +47,35 @@ reviewed.
 
 ### Revamp Slick Code-generator
 There is a long list of smaller and medium sized open tickets for the Slick code generator.
-Time to tackle them and give it another overhaul.
+Time to tackle them and give it another overhaul. This is going to be a very useful and somewhat easier project.
 
 https://github.com/slick/slick/issues?q=is%3Aopen+is%3Aissue+label%3Atopic%3Atype-providers
 
 Supervised by @cvogt and/or @szeiger
 
+### Slick bug and feature hunt
+There are lots of open tickets in the Slick issue tracker. Some which have long time fallen behind. This project would
+be about tackling as many of them as time permits. It will be a good chance to learn about the Slick code base
+and have an impact in this successful project. Some issues may be simple, but many issues may be challenging to fix and will require someone smart, dedicated and persistent enough to learn the details of how Slick works.
+
+https://github.com/slick/slick/issues
+
+Supervised by @cvogt and/or @szeiger
```

Comment:
Please link to your contact information page. I don't think that those are automatically linked to your github profile.